packages/@aws-cdk/assets/lib/fs/follow-mode.ts

```diff
@@ -1,3 +1,7 @@
+/**
+ * Symlink follow mode.
+ * @deprecated see `core.SymlinkFollowMode`
+ */
 export enum FollowMode {
```

Comment:
Why not simply type alias or extend `FollowMode`, `CopyOptions` and `FingerprintOptions` with new ones added in core?

Reply:
Aliases are not respected by jsii